src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -312,5 +312,32 @@ public void TestMailDeliveryAsync()
 
             Assert.True(task.Wait(1000));
```

Comment:
Sounds like a good suggestion (not sure about the common timeout though). Can you please file a 'test enhancement' issue?